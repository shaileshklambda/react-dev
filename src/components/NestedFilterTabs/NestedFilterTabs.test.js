import React from "react";
import renderer from "react-test-renderer";
import Nestedfiltertabs from ".";

it("Nestedfiltertabs: default", () => {
  const component = renderer.create(<Nestedfiltertabs />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
