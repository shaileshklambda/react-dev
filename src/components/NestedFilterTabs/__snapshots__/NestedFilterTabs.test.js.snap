// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nestedfiltertabs: default 1`] = `
<div>
  <div
    className="net_tab activeNested"
    id="controlled-tab-example"
  >
    <ul
      className="nav nav-tabs"
      role="tablist"
    />
    <div
      className="tab-content"
    />
  </div>
</div>
`;
